example/docker-compose.yaml

```diff
@@ -0,0 +1,55 @@
+version: '3.7'
```

Comment:
I think a quick readme.md with the few commands needed to get this up and running would be nice.

```
1. docker-compose up -d
2. Sign in to Grafana with admin/admin.
3. Anything else needed to setup Grafana.
```